koku/masu/database/sql/aws/reporting_aws_compute_summary_by_service_p.sql

```diff
@@ -59,6 +63,8 @@ INSERT INTO {{schema | sqlsafe}}.reporting_aws_compute_summary_by_service_p (
             SUM(blended_cost) AS blended_cost,
             SUM(savingsplan_effective_cost) AS savingsplan_effective_cost,
             SUM(markup_cost) AS markup_cost,
+            SUM(markup_cost_blended) AS markup_cost_blended,
+            SUM(markup_cost_savingsplan) AS markup_cost_savingsplan,
```

Comment:
For all instances of `markup_cost_blended` and `markup_cost_savingsplan` and `savingsplan_effective_cost` in the SQL files:
```suggestion
            SUM(coalesce(savingsplan_effective_cost, 0.0::numeric(33,15))) AS savingsplan_effective_cost,
            SUM(markup_cost) AS markup_cost,
            SUM(coalesce(markup_cost_blended, 0.0::numeric(33,15))) AS markup_cost_blended,
            SUM(coalesce(markup_cost_savingsplan, 0.0::numeric(33,15))) AS markup_cost_savingsplan,
```

Reply:
Updated for `markup_cost_blended`, `markup_cost_savingsplan`, and `savingsplan_effective_cost` 👍